libraries/Microsoft.Bot.Connector/Authentication/AuthenticationConstants.cs

```diff
@@ -10,6 +10,11 @@ namespace Microsoft.Bot.Connector.Authentication
     /// </summary>
     public static class AuthenticationConstants
     {
+        /// <summary>
+        /// Government Channel Service property value
+        /// </summary>
+        public const string GovernmentChannelService = "https://botframework.gov";
```

Comment:
The name here is unexpected. By having a "GovernmentChannelService" I expect to also see a "RegularChannelService" on [https://botframework.com](https://botframework.com). Likewise the ".gov" vs ".us" URIs, all with with "Government" in the name is additionally confusing. 

Is there a way to simplify this?

Reply:
I will change the constant to have a .us ending.

I think I'll move all the government constants to its own class and same with the enterprise one.